tests/debugger/do_src.expected

```diff
@@ -9,14 +9,16 @@ Stopped at tests/debugger/do_src.js:20
 (jerry-debugger) s
 Stopped at <unknown>:1
 (jerry-debugger) src
-f = function f() {
+
+1   > f = function f() {
```

Comment:
Why there is an empty line before this? We also need tests for src with argument and the new display commands.